policies.md

```diff
@@ -6,53 +6,108 @@ gcam-version: v5.1
 
 One of GCAM's uses is to explore the implications of different future policies. There are a number of types of policies that can be easily modeled in GCAM. The most common of these are discussed below.
 
-## Emissions-Related Policies
-There are three main policy approaches that can be applied in GCAM to reduce emissions of CO2 or other greenhouse gases: carbon or GHG prices, emissions constraints, or climate constraints. In all cases, GCAM implements the policy approach by placing a price on emissions. This price then filters down through all the systems in GCAM and alters production and demand. For exammple,a price on carbon would put a cost on emitting fossil fuels. This cost would then influence the cost of producing electricity from fossil-fired power plants that emit CO2, which would then influence their relative cost compared to other electricity generating technologies and increase the price of electricity. The increased price of electricity would then make its way to consumers that use electrcity, decreasing its competitiveness relative to other fuels and leading to a decrease in electricity demand. The three policy approaches are described below.
+Table Of Contents
+
+* [Emissions-Related Policies](#emissions-policies)
+    * [Linked Emission Markets](#linked-markets)
+    * [Markets For non-CO<sub>2</sub> Emission Species](#non-co2-markets)
+* [Land-Use Policies](#land-use-policies)
```

Comment:
For some reason it renders with a break between this bullet and the previously indented ones.  I'm not sure how to fix that though..